.github/workflows/pull-request.yml

```
@@ -65,11 +65,22 @@ jobs:
         run: |
           git clone git@github.com:MrSquaare/private-repository.git
 
+      - name: Run tests for GitHub
+        env:
+          OUTPUT_SSH_PATH: ${{ steps.github.outputs.ssh-path }}
```

actionlint (1.7.7)

property "ssh-path" is not defined in object type {}

(expression)